bin/_ci_config.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# _docker_run: wrap docker run command
+#
+# usage: _docker_run [COMMAND]
+#
+#   COMMAND: command to be run in Arnold container (_e.g._ ansible-playbook deploy.yml)
+#
+# purpose:
+#
+# This utility improves the developer experience by dynamically setting
+# environment variables required to run ansible playbooks and play with
+# an OpenShift instance from Arnold's container (see Dockerfile).
+function _docker_run() {
```

Comment:
why not reusing the function in `_config.sh` file ? This function is configurable, you can change the `env_file` used and the `TTY` option.

Reply:
Yes, but we also mount sources as a volume, I wanted to avoid making the original function too complicated.

Reply:
I've made the `docker_run` command more complex to use as you asked :trollface: